releng/MWE.setup

```diff
@@ -0,0 +1,220 @@
+      targetURL="${workspace.location|uri}/.metadata/.plugins/org.eclipse.jdt.ui/dialog_settings.xml"
+      encoding="UTF-8"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.orbit"
+      value="http://download.eclipse.org/tools/orbit/downloads/drops/R20170919201930/repository"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="xtext.version"
+      defaultValue="releases"
+      label="Xtext Version">
+    <choice
+        value="latest"
+        label="Latest Successful Build"/>
+    <choice
+        value="milestones"
+        label="Latest Milestone"/>
+    <choice
+        value="releases"
+        label="Latest Release"/>
+  </setupTask>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/${xtext.version}"
+      defaultValue=""
+      label="Xtext p2 Respository URL"/>
+  <setupTask
+      xsi:type="git:GitCloneTask"
+      id="git.clone.emf.mwe"
+      remoteURI="emf/org.eclipse.mwe"
+      pushURI="">
+    <annotation
+        source="http://www.eclipse.org/oomph/setup/InducedChoices">
+      <detail
+          key="inherit">
+        <value>eclipse.git.remoteURIs</value>
+      </detail>
+      <detail
+          key="label">
+        <value>MWE Git or Gerrit Repository</value>
+      </detail>
+      <detail
+          key="target">
+        <value>remoteURI</value>
+      </detail>
+    </annotation>
+    <configSections
+        name="gerrit">
+      <properties
+          key="createchangeid"
+          value="true"/>
+    </configSections>
+    <description>MWE Git Repository</description>
+  </setupTask>
+  <setupTask
+      xsi:type="setup.targlets:TargletTask">
+    <targlet
+        name="MWE"
+        activeRepositoryList="${eclipse.target.platform}">
+      <requirement
+          name="*"/>
+      <requirement
+          name="org.eclipse.platform.feature.group"/>
+      <sourceLocator
+          rootFolder="${git.clone.emf.mwe.location}"
+          locateNestedProjects="true"/>
+      <repositoryList
+          name="Photon">
+        <repository
+            url="http://download.eclipse.org/eclipse/updates/4.8-I-builds"/>
```

Comment:
Update to `http://download.eclipse.org/eclipse/updates/4.8`

Reply:
done